beta-src/src/components/map/components/WDArrowContainer.tsx

```diff
@@ -14,6 +14,7 @@ import ArrowColor from "../../../enums/ArrowColor";
 import drawArrowFunctional, {
   getTargetXYWH,
   getArrowX1Y1X2Y2,
+  makeSVGDrawAsUnsavedAnimateElement,
```

Comment:
This name doesn't feel like it's as clear as it could be. Can it be more about what it is for, not how it does it?